sklearn/feature_selection/tests/test_feature_select.py

```diff
@@ -402,36 +405,44 @@ def test_select_fdr_regression():
     Test that fdr heuristic actually has low FDR.
     """
     def single_fdr(alpha, n_informative, random_state):
-        n_features = 5 * n_informative
-
-        X, y = make_regression(n_samples=150, n_features=n_features,
+        X, y = make_regression(n_samples=150, n_features=20,
```

Comment:
It seemed a more interesting thing to test. The relative amount of informative feature does not have to be always the same.